### PR TITLE
Add current and expected jsons with transformer for edx.course.student_notes.searched

### PR DESCRIPTION
**Pull request by HamzaFarooqArbisoft**
**Trello Link:** _https://trello.com/c/NvpxudQH/118-teams-related-event-edxteamsearched_

**Description:** _When a user performs a search for teams from the topic view under the Teams page of the courseware, the server emits an edx.team.searched event._

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/transformers/notes_transformers.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ def edx_course_student_notes_added(current_event, caliper_event):
 
 def edx_course_student_notes_viewed(current_event, caliper_event):
     """
-    This event is generated when the user presses the previous button to traverse
-    over previous unit of same chapter.
+    The server emits an edx.course.student_notes.added event when the student
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/notes_transformers.py]: Please use the correct doc string.

File: openedx/features/caliper_tracking/transformers/notes_transformers.py [hunk 2]
```diff
@@ -162,3 +163,45 @@ def edx_course_student_notes_deleted(current_event, caliper_event):
     })
 
     return caliper_event
+
+
+def edx_course_student_notes_searched(current_event, caliper_event):
+    """
+    The server emits an edx.course.student_notes.added event when the student
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/notes_transformers.py]: Please use the correct doc string.

File: openedx/features/caliper_tracking/transformers/notes_transformers.py (hunk 1 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/transformers/notes_transformers.py]: done

File: openedx/features/caliper_tracking/transformers/notes_transformers.py (hunk 2 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/transformers/notes_transformers.py]: done

**Pull request merged**